grammars/asciidoc.cson

```diff
@@ -30,7 +30,7 @@
   #
   #   === Level 2 Section
   {
-    'begin': '^={1,6}\\s+'
+    'begin': '^={1,6}\\p{Blank}+'
```

Comment:
`\s` matches both spaces and tabs as well. However `\s` also matches new lines, but I don't see what issues it may cause.

Anyway it was mostly out of curiosity I'm fine with `\p{Blank}`.